<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_qt_charactercreation</class>
 <widget class="QWidget" name="client_qt_charactercreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <widget class="QPushButton" name="createPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="backPushButton">
     <property name="text">
      <string>Atras</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Crea un nuevo personaje 
e ingresa al juego</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clase:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedClassLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Raza:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedRaceLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="humanRButton">
         <property name="text">
          <string>Humano</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="elfRButton">
         <property name="text">
          <string>Elfo</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dwarfRButton">
         <property name="text">
          <string>Enano</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gnomeRButton">
         <property name="text">
          <string>Gnomo</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">raceButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="mageRButton">
         <property name="text">
          <string>Mago</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">classButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="clericRButton">
         <property name="text">
          <string>Clerigo</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">classButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="paladinRButton">
         <property name="text">
          <string>Paladin</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">classButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="warriorRButton">
         <property name="text">
          <string>Guerrero</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">classButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Seleccione una Clase</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Seleccione una Raza</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Vuelve a la pantalla de
inicio de sesion</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="raceButtonGroup"/>
  <buttongroup name="classButtonGroup"/>
 </buttongroups>
</ui>
